tee-worker/identity/litentry/core/native-task/receiver/src/types.rs

```diff
@@ -123,8 +135,5 @@ pub enum NativeTaskError {
 	ExtrinsicConstructionFailed(String), // Stringified itp_extrinsics_factory::Error
 	ExtrinsicSendingFailed(String),      // Stringified sgx_status_t
 	InvalidRequest,
-}
-
-pub enum NativeRequest {
-	// TODO: Define the tasks
```

Comment:
I see this is commented out, does it mean we are re-using TrustedCall enum for now?

Reply:
Initially this was the equivalent of `Web2IdentityVerificationRequest`, but since we don't need to add the web2 accounts in two steps anymore it's not needed. Yes, for now we are re-using  the`TrustedCall` enum for the native tasks, as we do for `request_vc`

Reply:
I see - it's fine for now

But let's keep in mind that in the future we might need them different names or enum types

Reply:
Yeah, we need to do some clean up when doing the migration or when unifying the api